### PR TITLE
[RW-5622][risk=no] allow for null answer concept id

### PR DESCRIPTION
**Pull request by freemabd**
allow for null answer concept id

### CODE REVIEW

File: api/src/main/java/org/pmiops/workbench/cdr/dao/CBCriteriaDao.java [hunk 1]
```diff
@@ -312,7 +297,7 @@ Long findQuestionCountByDomainAndIdAndTerm(
               + "join cb_survey_attribute csa on csv.survey_id = csa.survey_id "
               + "where csv.concept_id = :surveyConceptId "
               + "and csa.question_concept_id = :questionConceptId "
-              + "and csa.answer_concept_id = :answerConceptId "
+              + "and (csa.answer_concept_id = :answerConceptId or :answerConceptId is null)"
```
Comment (jscherdin) [inline api/src/main/java/org/pmiops/workbench/cdr/dao/CBCriteriaDao.java:300]: add a comment to explain this

**Review by jscherdin — CHANGES_REQUESTED** [review]
just the one issue. otherwise good to go

File: api/src/main/java/org/pmiops/workbench/cdr/dao/CBCriteriaDao.java (hunk 1 above)
Comment (freemabd) [reply api/src/main/java/org/pmiops/workbench/cdr/dao/CBCriteriaDao.java]: done.

**Review by jscherdin — APPROVED** [review]
LGTM

**Pull request merged**